zlp-scheduler/db/schema.rb

```diff
@@ -15,6 +15,16 @@
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
 
+  create_table "adminstudrecs", force: :cascade do |t|
```

Comment:
I think that Austin's PR eliminated this. Could you check that you have the changes to the migrations pulled to your fork?

Reply:
That's weird because the last time I check my fork is up to date with the master.

Reply:
When you ran the migrations did you completely drop the tables and run the migrations?